components/tidb_query/src/expr_util/collation/utf8mb4.rs

```diff
@@ -58,17 +58,277 @@ const GENERAL_CI_PLANE_01: [u16; 256] = [
     0x00C6, 0x00C6, 0x00D8, 0x00D8,
 ];
 
+const GENERAL_CI_PLANE_02: [u16; 256] = [
```

Comment:
Should the table be `static` instead of `const`? I'm afraid these tables may be copied to the stack where they are used...

Reply:
done